exampleFolder/index.js

```diff
@@ -39,7 +39,7 @@ app.delete('/deletePerson/:id', (req, res) => {
 //GET Route (/getAllPeople)
 app.get('/getAllPeople', (req, res) => {
     axios.get('http://java-sample-api-2020.herokuapp.com/getAllPeople')
-    .then((Response) => {
+    .then((response) => {
         console.log(response)
```

Comment:
On line 42, in the .then body, all that is missing is the response back to postman. If left missing, postman will to hang on loading screen.